src/groups/mqb/mqbnet/mqbnet_channel.cpp

```diff
@@ -63,17 +63,18 @@ void Channel::Stats::reset()
 // -------------
 
 Channel::Channel(bdlbb::BlobBufferFactory* blobBufferFactory,
-                 ItemPool*                 itemPool,
                  const bsl::string&        name,
                  bslma::Allocator*         allocator)
 : d_allocators(allocator)
-, d_allocator_p(d_allocators.get(bsl::string("Channel-") + name))
+, d_allocator_p(d_allocators.get("Channel"))
```

Comment:
@dorjesinpo, actually, there are 2 allocators used (`Channel` and `ItemPool`), and I will add more in the next PR.